### ISSUE TITLE
Sub-chapters: cannot detect chapters with "."

### ISSUE THREAD

**Issue by Brototype** [jneidel/mangareader-dl#5]
Example: https://mangalife.us/manga/Battle-Angel-Alita has chapters that include a `.`
```
Fight 51.7
Fight 51.6
Fight 51.5
Fight 51
Fight 50
...
```
it detects every chapter up to 51 perfectly fine but I could not figure out how to configure it for chapters containing `.` characters

**Comment by jneidel** [comment]
Yeah, sub-chapters are not supported as I personally wouldn't use them.

But for some series on mangalife it is necessary... ([another example](https://mangalife.us/manga/Mob-Psycho100))

I'll take a look at it in the future.
